### ISSUE TITLE
PipelineReaderJSON unable to parse pipelines created during translate

### ISSUE THREAD

**Issue by chambbj** [PDAL/PDAL#1221]
```console
$ pdal translate 1.2-with-color.las output.laz range --filters.range.limits=Z[400:500] -p foo.json
$ pdal pipeline foo.json
PDAL: PipelineReaderJSON: unable to process pipeline file "foo.json". JSON is invalid.
```

**Comment by abellgithub** [comment]
Well, the output is valid JSON, so the error is misleading.  I've fixed that, but the bottom line is that the output matches the old XML output and doesn't exactly match the new JSON spec.  Should it?  I guess it depends on the purpose of the `-p` option on `pdal translate` and I'm not sure I know.  @hobu?

**Comment by chambbj** [comment]
Oh, I was going to take a look at this. I agree, calling the JSON invalid was extreme.

IMO, the JSON out should match the spec. Maybe it has more detail than the pipeline reader needs (e.g., execution metadata), but the reader should gracefully ignore all of that.

**Comment by abellgithub** [comment]
Closed via: c29f692827ba192ffe398524b610eea9e3d90750
